spec/component/importers/activity_insight_importer_spec.rb

```diff
@@ -4907,9 +4914,9 @@
 
         it 'logs errors to ImporterErrorLog' do
           if import == 'Publication'
-            expect { importer.call }.to change(ImporterErrorLog, :count).by 5
+            expect { importer.call }.to change(ImporterErrorLog, :count).by 6
           else
-            expect { importer.call }.to change(ImporterErrorLog, :count).by 2
+            expect { importer.call }.to change(ImporterErrorLog, :count).by 5
```

Comment:
I'm having trouble trying to understand why this went from 2 -> 5?  I can understand the one above it since we added another publication to the fixture, but I feel like this one shouldn't have changed.  I may not be considering something though.  Any ideas why this changed?

Reply:
I think maybe because an error is being raised somewhere we aren't aware of?  But the only thing that has changed is the DOIVerificationJob is being invoked.

Reply:
Good catch, @ajkiessl! I didn't notice this. Is there a corresponding decrease in the number of records being successfully imported somewhere?

Reply:
I popped a byebug in the error rescue for the publication part of the import and the error is a `<JSON::ParserError: unexpected token at ''>`.  It's being risen from `DoiVerificationJob.new.perform(pub_record)`.  Not sure if that's from something inside the job, or if delayed job is failing for some other reason.  I thought these jobs would be queued and ran in a separate process? But maybe delayed job works differently in the test environment and runs the process synchronously.

Reply:
It seems like it's a problem with the spec and how the HttpService is mocked. Eric & I met, and we're going to just mock `DoiVerificationJob` instead of trying to return an appropriate response for each http request. Then we can also use that to check that the job was called (for now until we decide when/if we want to remove it from the importer)